Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /code
+WORKDIR /code
```

Comment:
What about using /opt/scancode.io/ instead?

Reply:
/code is the location suggested by the docs https://docs.docker.com/compose/django/#define-the-project-components
Let's try /opt/scancode.io/ though